app/services/support_interface/start_of_cycle_notifier.rb

```diff
@@ -0,0 +1,18 @@
+module SupportInterface
+  class StartOfCycleNotifier
+    def call(service:, batch_size: 500, year: 2022)
```

Comment:
maybe we should be calculating the batch size from the hours/provider scope at the time. For instance we have 90 providers and 9 hours left, we send the email to the users of the 10 providers. Then when this runs on the next hour we have 80 and 8 hours etc. let me know your thoughts